src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -42,6 +41,12 @@ public MainWindow(PowerToysRunSettings settings, MainViewModel mainVM)
             _firstDeleteTimer.Interval = 1000;
         }
 
+        protected override void OnSourceInitialized(EventArgs e)
+        {
+            base.OnSourceInitialized(e);
+            WindowsInteropHelper.SetPopupStyle(this);
```

Comment:
FZ could intercept window creation before we have a chance to set its style to popup.
To test this, manually add an entry to `app-zone-history.json`. Sometimes, the Launcher window will not appear.